caas/caffe_worker/Worker-remote

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+#
+# Autogenerated by Thrift Compiler (0.9.3)
+#
+# DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
+#
+#  options string: py
+#
+
+import sys
+import pprint
+from urlparse import urlparse
+from thrift.transport import TTransport
+from thrift.transport import TSocket
+from thrift.transport import TSSLSocket
+from thrift.transport import THttpClient
+from thrift.protocol import TBinaryProtocol
+
+from caffe_worker import Worker
+from caffe_worker.ttypes import *
+
+if len(sys.argv) <= 1 or sys.argv[1] == '--help':
+  print('')
+  print('Usage: ' + sys.argv[0] + ' [-h host[:port]] [-u url] [-f[ramed]] [-s[sl]] function [arg1 [arg2...]]')
+  print('')
+  print('Functions:')
+  print('  BaseOutput status()')
+  print('  BaseOutput commission(CommissionInput ci)')
+  print('  BaseOutput decommission()')
+  print('  PredictionOutput predict(PredictionInput pi)')
+  print('')
+  sys.exit(0)
+
+pp = pprint.PrettyPrinter(indent = 2)
```

Comment:
That is how it was done initially. We have done this intentionally for the users to get started easily. Otherwise, user has to install thrift and generate files and he has to be aware of the directory and has to setup the sys path accordingly. This is not going to change too frequently.